docs/source/en/quantization.md

```diff
@@ -468,6 +468,10 @@ Try 4-bit quantization in this [notebook](https://colab.research.google.com/driv
 
 This section explores some of the specific features of 4-bit models, such as changing the compute data type, using the Normal Float 4 (NF4) data type, and using nested quantization.
 
+#### Push 4-bit models on the Hub
+
+If you have `bitsandbytes>=0.41.3`, you can serialize 4-bit models and push them on Hugging Face Hub. Simply call `model.push_to_hub()` after loading it in 4-bit precision.
```

Comment:
Thanks for the update! I think it would be nicer to include this [here](https://github.com/huggingface/transformers/blob/260b9d2179ea9592b24ff102ab9ea672f6a4f3ef/docs/source/en/quantization.md?plain=1#L348) so it parallels the 8-bit section :)

Reply:
Done, maybe it is a bit repetitive, lmk what do you think!

Reply:
Okay , done!